src/components/tooltip/tooltip.tsx

```diff
@@ -114,6 +114,7 @@ export class Tooltip {
         return (
             <div class="trigger-anchor">
                 <limel-portal
+                    openDirection="bottom-start"
```

Comment:
First we wanted to make this an editable `prop` and introduce a new feature, and default it to `top`. But due to [this bug](https://github.com/Lundalogik/lime-elements/issues/1408) we cannot this this yet. So we will have to do that later.

Reply:
Oh, _there_ is the issue I was talking about in our call @LucyChyzhova ☝️ 😄

So, not directly related to limel-portal, but to the specific case where we use it in limel-tooltip. That's why I couldn't quite remember what the problem was, or find the issue 😄